lib/voice/VoiceConnection.js

```diff
@@ -200,7 +200,7 @@ class VoiceConnection extends EventEmitter {
                 this.connectionTimeout = null;
             }
             this.sendWS(VoiceOPCodes.IDENTIFY, {
-                server_id: this.id === "call" ? data.channel_id : this.id,
+                server_id: this.id === this.id,
```

Comment:
noticed a tiny oversight here:
```suggestion
                server_id: this.id,
```

Reply:
I must've been on something with making the pr I did to remove half ofthis to miss something so obvious